apps/code-examples/src/app/code-examples/lookup/search/basic/demo.component.spec.ts

```diff
@@ -0,0 +1,95 @@
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { SkyMediaBreakpoints, SkyMediaQueryService } from '@skyux/core';
+import { MockSkyMediaQueryService } from '@skyux/core/testing';
+import { SkySearchComponent } from '@skyux/lookup';
```

Comment:
🤔 We export the search component without a lambda because of an issue with list builder; however, the consumer should never use it directly. Is there a way to refactor the unit test to not require it?

Reply:
i can remove that import if i remove the unit test showing consumers how to shrink their window size in unit tests which is needed for some of the harness functions, do we think we're ok not showing them that?

Reply:
If I remove the `overrideComponent` call the tests still pass... Is it needed? 🤷‍♂️